src/Language/PureScript/Make.hs

```diff
@@ -397,3 +402,34 @@ buildMakeActions outputDir filePathMap foreigns usePrefix =
 
   progress :: ProgressMessage -> Make ()
   progress = liftIO . putStrLn . renderProgressMessage
+
+-- | For a given CoreFn module and corresponding FFI module file path,
+-- check that there is exactly one FFI declaration for each
+checkForeignDecls :: CF.Module CF.Ann -> FilePath -> SupplyT Make ()
```

Comment:
Could perhaps leave `Module` polymorphic here, I don't think knowing about the annotations is necessary in here, is it?